gluoncv/model_zoo/fcn.py

```diff
@@ -111,16 +111,14 @@ def get_fcn(dataset='pascal_voc', backbone='resnet50', pretrained=False,
     from ..data.ade20k.segmentation import ADE20KSegmentation
     acronyms = {
         'pascal_voc': 'voc',
-        'pascal_aug': 'voc',
         'ade20k': 'ade',
     }
     datasets = {
         'pascal_voc': VOCSegmentation,
-        'pascal_aug': VOCAugSegmentation,
         'ade20k': ADE20KSegmentation,
     }
     # infer number of classes
-    model = FCN(datasets[dataset].NUM_CLASS, backbone=backbone, ctx=ctx, **kwargs)
+    model = FCN(datasets[dataset].NUM_CLASS, backbone=backbone, pretrained=pretrained, ctx=ctx, **kwargs)
```

Comment:
This `pretrained` arg has different meaning. It means the FCN is pretrained on the segmentation dataset

Reply:
pretrained here stands for using weights for the entire FCN network, I think what you want to do is add an option `pretrained_base=True` and pass to FCN instead

Reply:
agree with @zhreshold

Reply:
Fixed.